dvc/repo/experiments/queue/base.py

```diff
@@ -147,6 +147,38 @@ def clear(self) -> List[str]:
         self._remove_revs(stash_revs)
         return removed
 
+    def status(self) -> List[Mapping[str, Optional[str]]]:
+        """Show the status of exp tasks in queue"""
+        from datetime import datetime
+
+        result: List[Mapping[str, Optional[str]]] = []
+
+        def _get_timestamp(rev):
+            commit = self.scm.resolve_commit(rev)
+            return datetime.fromtimestamp(commit.commit_time)
+
+        for queue_entry in self.iter_active():
+            result.append(
+                {
+                    "Rev": queue_entry.stash_rev,
+                    "Name": queue_entry.name,
+                    "timestamp": _get_timestamp(queue_entry.stash_rev),
+                    "Status": "Running",
```

Comment:
For internal APIs (and not UI) these keys should all be lowercased for consistency